activerecord/lib/active_record/integration.rb

```diff
@@ -161,8 +161,8 @@ def to_param(method_name = nil)
         end
       end
 
-      def collection_cache_key(collection = all, timestamp_column = :updated_at) # :nodoc:
-        collection.send(:compute_cache_key, timestamp_column)
+      def collection_cache_key(collection = all, id_column: :id, timestamp_column: :updated_at) # :nodoc:
+        collection.send(:compute_cache_key, { id_column: id_column, timestamp_column: timestamp_column })
```

Comment:
Can we drop the explicit Hash allocation here (i.e. just pass the keyword args)?

Reply:
Good catch, I've rebased accordingly.